src/actinia_core/resources/ephemeral_processing_with_export.py

```diff
@@ -261,6 +275,14 @@ def _export_raster(self, raster_name,
         self._update_num_of_steps(1)
         self._run_module(p)
 
+        # DELETE ONCE GDAL 3.2 HAS BEEN RELEASED
```

Comment:
Maybe add a TODO. Should the overview than be set in line 264? Or do we have to test it?

Reply:
COG sets overviews automatically if not specified differently. Both blocks can be deleted once the GDAL bug is fixed.